analyses/cell-type-wilms-tumor-06/00_run_workflow.sh

```diff
@@ -39,6 +39,13 @@ data_dir="../../data/current"
 notebook_template_dir="notebook_template"
 notebook_output_dir="notebook"
 
+# Define test data string to use with 06_infercnv.R
+if [[ $TESTING -eq 1 ]]; then
```

Comment:
```suggestion
if [[ $IS_CI -eq 1 ]]; then
```

Reply:
But maybe you just want to do the following at line 28 to be consistent with the other variables and shell script:

```sh
TESTING=${TESTING:-0}
```

Don't forget to change lines 136 and 174 (at least), too, if you go this way

Reply:
Ended up going with  `TESTING=${TESTING:-0}` and updating other lines accordingly.